scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -398,12 +432,14 @@ task ExtractFilterTask {
   }
 
   String intervals_name = basename(intervals)
-
+  File monitoring_script = "gs://gvs_quickstart_storage/cromwell_monitoring_script.sh"
```

Comment:
A bucket with `quickstart` in its name might not be the best place for a script that's going to be used for non-quickstart runs. Maybe `gs://gvs_internal`?